code_search/src/code_search/t2t/query.py

```diff
@@ -14,16 +14,19 @@ def get_encoder(problem_name, data_dir):
   return problem.feature_info["inputs"].encoder
 
 
-def encode_query(encoder, query_str):
+def encode_query(encoder, query_str, embed_code=False):
```

Comment:
Can we get rid of embed_code now that we are concatenating the two vectors and computing both embeddings?

Reply:
This actually leads to a bunch of changes here and even the Dataflow step before this. Shall we defer this change to the final PR after we make sure the model works?

Reply:
Defer which change the whole PR or just removing embed_code?